requirements.in

```diff
@@ -0,0 +1,15 @@
+cosl
```

Comment:
I would pin these and let renovate handle dependencies for the sake of repeatable (or closer to repeatable) builds